### ISSUE TITLE
[BUG]  User getting error while unstaking GTC

### ISSUE THREAD

**Issue by jesuloluwao** [gitcoinco/passport#1304]
This user is getting an error while unstaking their GTC. I have tried all troubleshooting methods but they are getting this error 
![image](https://github.com/gitcoinco/passport/assets/88308638/fa2ba8b3-20cd-4c5d-815c-71ec6b970684)

Is it something you can help with?

 > <div dir="auto">Hi<div dir="auto"><br></div><div dir="auto">0x5b5b174827aEcf33882B502b160A75D9AF5E986E<br></div></div><br><div><div dir="ltr">вт, 23 мая 2023 г., 12:30 Lolu from Gitcoin &lt;<a href="mailto:support@gitcoin.co" rel="nofollow" target="_blank">support@gitcoin.co</a>&gt;:<br></div><blockquote style="margin:0 0 0 .8ex;border-left:1px #ccc solid;padding-left:1ex"><u></u> <div style="background-color:white"> <div style="color:transparent;font-size:0px;border:0;width:1px;margin:0px;padding:0px;border-width:0px!important;display:none!important;line-height:0px!important"></div> <table cellspacing="0" border="0" cellpadding="0" align="center" style="font-family:Helvetica,Arial,sans-serif;letter-spacing:0;width:100%;border-spacing:0;border-collapse:separate;table-layout:fixed"> <tbody> <tr> <td style="font-family:Helvetica,Arial,sans-serif;padding:0"> </td> </tr> <tr> <td style="font-family:Helvetica,Arial,sans-serif;line-height:20px;padding:0"> <div style="margin-top:0;padding-top:

**Link:** [Conversation with sanik.b78@gmail.com](https://app.intercom.com/a/apps/xaafeyri/inbox/inbox/all/conversations/49122)

**Customer:** [sanik.b78@gmail.com](https://app.intercom.com/a/apps/xaafeyri/users/64350a0e6d7d682006a5c45b/all-conversations)
**Customer ID:** 115f34a2-a34d-4151-96b2-37a3420ad82f
___
Intercom Operator will send a note to [this conversation](https://app.intercom.com/a/apps/xaafeyri/inbox/inbox/all/conversations/49122) when this issue is closed.

**Comment by lucianHymer** [comment]
@Jeremy-Gitcoin @erichfi  This is ready for testing! I've confirmed that this specific issue is fixed, it's fairly technical to test.

But I did make substantial code changes, so it'd be great if one of you guys could just try staking (both on yourself and other addresses) to make sure things work as expected. Updating community stake should now work (it was broken before).

It's available on the review server here: https://main.dshoxaxrgsb58.amplifyapp.com/StakeDashboard

This points to a Sepolia contract. To get some fake sepolia GTC, just use the `mint` function here. https://sepolia.etherscan.io/address/0x0E3efD5BE54CC0f4C64e0D186b0af4b7F2A0e95F#writeContract
This will send 100 fake GTC to your address.

The end date of the round is "invalid date", this is to be expected. I initiated a round that basically doesn't end.
